### PR TITLE
GOK-229 | add. save functionality to dispense drugs and mark it as dispensed

### PR DESCRIPTION
**Pull request by abinaya-u**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more redable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code
- [ ] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Summary
This PR contains changes related to dispense prescribed drugs for an active patient and mark that specific drug as dispensed. Also to notify user with appropriate notifications. 

## Screenshots
image

### CODE REVIEW

File: src/service/bahmni-encounter.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { dispenseConceptURL, getRequest, postRequest, sessionURL } from "../utils/api-utils";
+
+const bahmniEncounterPost = async (data, location) => {
+	const dispenseConceptData = await getRequest(dispenseConceptURL);
+	const session = await getRequest(sessionURL)
+	const observations = data.dispense_drugs.map((drug) => ({
+		providers: [
+			{
```
Comment (mohan-13) [inline src/service/bahmni-encounter.js]: Why is this needed? Provider information is not needed at observation level right ?

Comment (abinaya-u) [reply src/service/bahmni-encounter.js]: yeah, observation doesn't require providers details.. will remove it

Comment (abinaya-u) [reply src/service/bahmni-encounter.js]: removed

**Review by mohan-13 — APPROVED** [review]
